### PR TITLE
Add barcodes for texts

### PR DESCRIPTION
**Pull request by FranBonath**

### CODE REVIEW

File: status/barcode.py [hunk 1]
```diff
@@ -112,21 +115,23 @@ def make_barcode(label, print_bc):
     formattedLabel.append('^LH0,0') # label home position (label home = LH)
     if print_bc:
         ch_size = '20'
-        xpositionText = '360'
+        xpositionText = '360' 
+        textHight = '50'
```
Comment (aanil) [inline status/barcode.py:119]:
```suggestion
        textHeight = '50'
```

File: run_dir/design/barcode.html [hunk 2]
```diff
@@ -45,6 +41,14 @@ <h3><span>Label and barcode printing for workset plates or internal use</span></
         <button type="submit" class="btn btn-primary btn-lg" data-script="barcode">Print</button>
       </div>
     </div>
+    <div class="row text-center" style="margin-right:5px">
```
Comment (aanil) [inline run_dir/design/barcode.html:44]: Consider changing to bootstrap margin classes

File: status/barcode.py [hunk 3]
```diff
@@ -49,7 +49,10 @@ def post(self):
                         print_barcode(linesToPrint)
                 else: # file submitted is a text file
                     for line in linesToPrint.splitlines(): # split into the different lines of the text file
-                        createdLabel = make_barcode(line, match_barcode(line, False))
+                        if self.get_argument('print_with_barcode', '') : # is true when barcode box is ticked
```
Comment (aanil) [inline status/barcode.py:52]: If `print_with_barcode`  is boolean, why not just pass it to `make_barcode` like so
`make_barcode(line, match_barcode(line, self.get_argument('print_with_barcode', False))`

File: status/barcode.py [hunk 4]
```diff
@@ -112,21 +115,23 @@ def make_barcode(label, print_bc):
...
         ch_size = '20'
-        xpositionText = '360'
+        xpositionText = '360' 
+        textHight = '50'
         if len(label) > 9:
             ch_size = '10' # squeezes the text for long texts
             xpositionText = '440' # moves the text position because the bc is longer
-        formattedLabel.append('^FO{0},35^AFN,60,{1}^FN1^FS'.format(xpositionText, ch_size)) # AF = assign font F, field number 1 (FN1), print text at position field origin (FO) rel. to home
-        formattedLabel.append('^FO80,35^BCN,70,N,N^FN2^FS') # BC=barcode 128, field number 2, normal orientation, height 70, no interpretation line.
+            textHight = '38'
```
Comment (alneberg) [inline status/barcode.py:123]: Same here? But this cooould be a weird ZPL command?

File: status/barcode.py [hunk 5]
```diff
@@ -112,21 +115,23 @@ def make_barcode(label, print_bc):
...
         ch_size = '20'
-        xpositionText = '360'
+        xpositionText = '360' 
+        textHeight = '50'
         if len(label) > 9:
             ch_size = '10' # squeezes the text for long texts
             xpositionText = '440' # moves the text position because the bc is longer
-        formattedLabel.append('^FO{0},35^AFN,60,{1}^FN1^FS'.format(xpositionText, ch_size)) # AF = assign font F, field number 1 (FN1), print text at position field origin (FO) rel. to home
-        formattedLabel.append('^FO80,35^BCN,70,N,N^FN2^FS') # BC=barcode 128, field number 2, normal orientation, height 70, no interpretation line.
+            textHight = '38'
```
Comment (alneberg) [inline status/barcode.py:123]:
```suggestion
            textHeight = '38'
```

File: status/barcode.py [hunk 6]
```diff
@@ -112,21 +115,23 @@ def make_barcode(label, print_bc):
...
-        xpositionText = '360'
+        xpositionText = '360' 
+        textHeight = '50'
         if len(label) > 9:
             ch_size = '10' # squeezes the text for long texts
             xpositionText = '440' # moves the text position because the bc is longer
-        formattedLabel.append('^FO{0},35^AFN,60,{1}^FN1^FS'.format(xpositionText, ch_size)) # AF = assign font F, field number 1 (FN1), print text at position field origin (FO) rel. to home
-        formattedLabel.append('^FO80,35^BCN,70,N,N^FN2^FS') # BC=barcode 128, field number 2, normal orientation, height 70, no interpretation line.
+            textHeight = '38'
+        formattedLabel.append('^FO{0},27^AFN,{1},{2}^FN1^FS'.format(xpositionText, textHight, ch_size)) # AF = assign font F, field number 1 (FN1), print text at position field origin (FO) rel. to home
```
Comment (alneberg) [inline status/barcode.py:124]:
```suggestion
        formattedLabel.append('^FO{0},27^AFN,{1},{2}^FN1^FS'.format(xpositionText, textHeight, ch_size)) # AF = assign font F, field number 1 (FN1), print text at position field origin (FO) rel. to home
```

File: status/barcode.py (hunk 3 above)
Comment (alneberg) [reply status/barcode.py:52]: It's fine the way it is as well

**Pull request merged**